### PR TITLE
Paginate list

### PR DESCRIPTION
**Pull request by andyw8**
This PR introduces the [Kaminari](https://github.com/amatsuda/kaminari) gem to handle pagination.

If there are less than 20 books on a list then the pagination navigation
is not shown.

Since the books index template is also used by the `FilterController`, I
have added pagination that too.

I wasn't keen on referring to the `Kaminari` class directly in the view,
since that would be a hard coupling, so I have wrapped that in a simple
helper.

### CODE REVIEW

File: app/controllers/books/filter_controller.rb [hunk 1]
```diff
@@ -1,8 +1,13 @@
 module Books
   class FilterController < ApplicationController
+    PER_PAGE = 20
```
Comment (andyw8) [inline app/controllers/books/filter_controller.rb]: I should probably extract this since it's the value as in `BooksController`.

Comment (taylorrf) [inline app/controllers/books/filter_controller.rb]: Maybe you can try to add a Kaminari default configs here: https://github.com/amatsuda/kaminari#general-configuration-options
and define the default number of "default_per_page" to 20. Doing it you will also be able to remove those ".per(PER_PAGE)" piece of code from both controllers.

Comment (andyw8) [inline app/controllers/books/filter_controller.rb]: Yeah, that sounds like a good idea.

**Comment by andyw8** [comment]
Updated to use Kaminari default page size config.

**Comment by taylorrf** [comment]
@andyw8 awesome! good job :+1:

**Pull request merged**